dxr/plugins/python/indexers.py

```diff
@@ -276,22 +277,16 @@ def analyze_tokens(self):
                 if tok_name in '([{':
                     paren_level += 1
                 elif tok_name in '}])':
-                    node_start, node_queue = paren_stack.pop(paren_level, (None, None))
-                    if node_start is not None:
-                        node_start_table[node_start] = node_queue
+                    paren_stack.pop(paren_level, (None, None))
```

Comment:
It doesn't matter what the 2nd param is here, since we're throwing it away, right? I'd rather in that case have something clearly unused, like `'dummy'`. Or catch and ignore the exception.